pkg/cmd/datamgr/cli/install/install.go

```diff
@@ -137,14 +143,21 @@ func NewCommand(f client.Factory) *cobra.Command {
 }
 
 func (o *InstallOptions) Run(c *cobra.Command, f client.Factory) error {
-	var resources *unstructured.UnstructuredList
-
-	// In case of Guest or Supervisor cluster, skip installing data manager
 	skipDataMgr := false
-	clusterFlavor, err := utils.GetClusterFlavor(nil)
-	if clusterFlavor == utils.TkgGuest || clusterFlavor == utils.Supervisor {
-		fmt.Printf("The Plugin is installed in %s\n. Skipping data manager installation.", clusterFlavor)
-		skipDataMgr = true
+	installCRDs := true
+	// Check if ItemActionPlugin feature is enabled.
+	if cmd.IsFeatureEnabled(o.Features, utils.VSphereItemActionPluginFlag) {
```

Comment:
Do we really need to check the feature flag when install data manager? The previous check based on the cluster flavor seems to be good enough.